RecoTracker/FinalTrackSelectors/plugins/TrackCutClassifier.cc

```diff
@@ -48,61 +66,157 @@ namespace {
   inline int nPixelHits(reco::Track const & tk) {
     return tk.hitPattern().numberOfValidPixelHits();
   }
-      
+  
   inline float dz(reco::Track const & trk, Point const & bestVertex) {
-      return std::abs(trk.dz(bestVertex));
+    return std::abs(trk.dz(bestVertex));
   }
   inline float dr(reco::Track const & trk, Point const & bestVertex) {
-      return std::abs(trk.dxy(bestVertex));
+    return std::abs(trk.dxy(bestVertex));
+  }
+
+  inline void dzCut_par1(reco::Track const & trk, int & nLayers, const float * par, float dzCut[]) {
+    float dzE =  trk.dzError();
+    for (int i=2; i>=0; --i) {
+      dzCut[i] = powN(par[i]*nLayers,4)*dzE;
+    }
+  }
+  inline void drCut_par1(reco::Track const & trk, int & nLayers, const float* par, float drCut[]) {
+    float drE =  trk.d0Error();
+    for (int i=2; i>=0; --i) {
+      drCut[i] = powN(par[i]*nLayers,4)*drE;
+    }
   }
+  
+  inline void dzCut_par2(reco::Track const & trk, int & nLayers, const float * par, float dzCut[]) {
+    float pt = float(trk.pt());
+    float eta = float(trk.eta());
 
+    // parametrized d0 resolution for the track pt
+    float nomd0E = sqrt(0.003*0.003+(0.001/pt)*(0.001/pt));
```

Comment:
Incidentally I was looking for the details of MultiTrackSelector in the context of phase1. In MultiTrackSelector these parameters were configurable (`res_par`), and `Phase1PU70` configuration indeed updates them, although the only difference to above is the use of `0.002` instead of `0.001` in some (but not all) places.

So while we're at it, would it make sense/be worth of the effort to keep these configurable? I don't claim these cuts would be used in the final phase1 tracking configuration, but in the mean time having them could be handy (especially to keep `Phase1PU70` alive as a reference if/when MultiTrackSelector gets removed, ref. @VinInn's comment https://github.com/cms-sw/cmssw/pull/12917/files#r49466613)